src/agoradatatools/etl/transform/biomarkers.py

```diff
@@ -0,0 +1,71 @@
+"""
+This module contains the transformation logic for the biomarkers dataset.
+This is for the Model AD project.
+"""
+
+import pandas as pd
+
+
+def transform_biomarkers(datasets: dict) -> list:
+    """
+    Takes dictionary of dataset DataFrames, extracts the biomarkers
+    DataFrame, and transforms it into a list of dictionaries grouped by
+    'model', 'type', 'ageDeath', 'tissue', and 'units'.
+
+    Args:
+        datasets (dict[str, pd.DataFrame]): dictionary of dataset names mapped to their DataFrame
+
+    Returns:
+        dict: a dictionary of biomarkers data modeled after intended final JSON structure
+    """
+    biomarkers_dataset = datasets["biomarkers"]
+
+    # Check that the dataset looks like what we expect
+    if not isinstance(biomarkers_dataset, pd.DataFrame):
+        raise ValueError("Biomarker dataset is not a pandas DataFrame")
```

Comment:
Should we add what the `biomarkers_dataset` is to the exception message?

Reply:
Yes we should! Thank you :)
I also changed the exception to TypeError because that is what it is.